tests/integration/base.py

```diff
@@ -30,6 +30,9 @@
 from dbt.events import AdapterLogger
 from dbt.contracts.graph.manifest import Manifest
 
+from ..utils import build_databricks_cluster_profile
```

Comment:
super nit: how about using absolute import?

Reply:
Good point!
Unfortunately, `pytest` doesn't recognize the absolute import for some reason.
I'll investigate it later, but let me merge this now to unblock other investigations.
Thanks.